tests/resources/image/README.md

```diff
@@ -0,0 +1,28 @@
+# Control Interface Tester workload
+
+The Control Interface Tester workload is used in system test to verify the correct execution of the Ankaios Control Interface.
+The workload reads a commands file, executes the commands and writes an output file containing the result of the commands.
+
+In case a Control Interface instance was not provided to the workload, a `NoAPI` result is written. If the Control Interface was preliminary closed by Ankaios, e.g., due to a protocol error, a `ConnectionClosed` result is provided.
+
+## Building ans pushing a new image
+
+It is planned to automate the process of building and pushing a new version of the container, but for now the process is done manually.
+
+To build a new image run the following command from the project root folder:
+
+```bash
+podman build -t ghcr.io/eclipse-ankaios/control_interface_tester:manual-build-<new version number> . -f tests/resources/image/Dockerfile
```

Comment:
Just for curiosity, do we need this as a multi-platform build?

Reply:
I also wondered, but for now it runs only on the amd64 machines. As this is the second time we needed to update the image, I just documented the current state as it was given to me.